src/OpenTelemetry/Internal/MathHelper.cs

```diff
@@ -15,6 +15,9 @@
 // </copyright>
 
 using System.Diagnostics;
+#if NETCOREAPP3_0_OR_GREATER
```

Comment:
I'm not sure if this should be changed to `NET6_0_OR_GREATER` or not, would you double check @ThomsonTan?

Reply:
The doc says it applies to .net core 3.0 and above. Is `NETCOREAPP3_0_OR_GREATER` sufficient to cover this? Or this repo only supports .NET 6 and above?

https://docs.microsoft.com/en-us/dotnet/api/system.numerics.bitoperations#applies-to

Reply:
Check this https://github.com/open-telemetry/opentelemetry-dotnet/blob/d1da1d66a3ab4369144e752b47f80dc1f63ae5b3/src/OpenTelemetry/OpenTelemetry.csproj#L4

Reply:
.net standard 2.0 includes .net core 2.0, 2.1, 2.2, 3.0, 3.1, 5.0, 6.0.  Sounds like `NETCOREAPP3_0_OR_GREATER` could work for .net standard 2.0, is this correct?

https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0#select-net-standard-version

Reply:
@ThomsonTan Not really. Given these targets...

`<TargetFrameworks>net6.0;netstandard2.1;netstandard2.0;net462</TargetFrameworks>`

...in the package `/lib` folder there will be dlls for each of those. Only `net6.0` would have this code.

If you wanted it to apply to the "net standards" as well as .NET Core 3+ you would need to do something like...

`NETCOREAPP3_0_OR_GREATER || NETSTANDARD2_0_OR_GREATER`

...but then it wouldn't compile because the API isn't actually available to netstandard APIs.

You could leave it as `NETCOREAPP3_0_OR_GREATER ` or switch it to `NET6_0_OR_GREATER` the result would be the same given the TFMs we are building for.

Reply:
@CodeBlanch thanks for the explanation. So in brief `NETCOREAPP3_0_OR_GREATER` is the same as `NET6_0_OR_GREATER` based on the target frameworks we are building? If so, I'd like to preserve the macro `NETCOREAPP3_0_OR_GREATER` because it can be directly attributed back to the doc on MSDN.

Reply:
@ThomsonTan Works for me! Merged 😄